docs/src/main/paradox/scala/http/routing-dsl/source-streaming-support.md

```diff
@@ -115,3 +115,14 @@ instances, which can be provided using any JSON marshalling library (such as Cir
 
 When implementing a custom support trait, one should simply extend the `EntityStreamingSupport` abstract class,
 and implement all of it's methods. It's best to use the existing implementations as a guideline.
+
+## Supporting custom content types
+
+In order to marshal into custom content types both a `Marshaller` that can handle that content type, 
+**as well as an `EntityStreamingSupport` of matching content type** is required.
```

Comment:
I _would_ like to, for inspiration, remark that the already existing streaming marshalling in ts-reaktive indeed provides a real `Marshaller[Source[T], RequestEntity]`, for [json](https://github.com/Tradeshift/ts-reaktive/blob/master/ts-reaktive-marshal-akka/src/main/java/com/tradeshift/reaktive/marshal/http/JSONMarshallers.java#L34) and [xml](https://github.com/Tradeshift/ts-reaktive/blob/master/ts-reaktive-marshal-akka/src/main/java/com/tradeshift/reaktive/marshal/http/XMLMarshallers.java#L44). No need for additional implicit framing types.